content/kubermatic/v2.18/architecture/requirements/support_policy/supported_versions/_index.en.md

```diff
@@ -44,18 +44,19 @@ these migrations.
 In the following table you can find the supported Kubernetes versions for the
 current KKP version.
 
-| KKP version | 1.22 | 1.21 | 1.20 | 1.19* | 1.18** | 1.17** |
-| ----------- | ---- | ---- | ---- | ----- | ------ | ------ |
-| 2.18.x      | ✓    | ✓    | ✓    | ✓     | -      | -      |
-| 2.17.x      | -    | ✓    | ✓    | ✓     | ✓      | -      |
-| 2.16.x      | -    | -    | ✓    | ✓     | ✓      | ✓      |
-| 2.15.x      | -    | -    | -    | ✓     | ✓      | ✓      |
-
-\* Kubernetes 1.17 and 1.18 releases have reached End-of-Life (EOL). We
+| KKP version | 1.22 | 1.21 | 1.20 | 1.19\* | 1.18\*\* | 1.17\*\* |
+| ----------- | ---- | ---- | ---- | ------ | -------- | -------- |
+| 2.18.x      | ✓    | ✓    | ✓    | ✓      | -        | -        |
+| 2.17.x      | -    | ✓    | ✓    | ✓      | ✓        | -        |
+| 2.16.x      | -    | -    | ✓    | ✓      | ✓        | ✓        |
+| 2.15.x      | -    | -    | -    | ✓      | ✓        | ✓        |
```

Comment:
I think this table is missing the entry 2.19

Reply:
These are the 2.18 docs. (See file path)